docs/i18n/ja/docusaurus-plugin-content-docs/current/getting-started/quickstart.md

```diff
@@ -74,7 +74,7 @@ Available options:
                            yourself.
   --persistence-dir DIR    The directory where the Hydra Head state is stored.Do
                            not edit these files manually!
-  --network-id INTEGER     Network identifier for a testnet to connect to. We
+  --testnet-magic INTEGER     Network identifier for a testnet to connect to. We
```

Comment:
Also here, `hydra-node --help > HERE`. Maybe we can automate this with in a workflow step?